neo4j/_async/io/_bolt.py

```diff
@@ -259,19 +259,20 @@ def get_handshake(cls):
         return b"".join(version.to_bytes() for version in offered_versions).ljust(16, b"\x00")
 
     @classmethod
-    async def ping(cls, address, *, timeout=None, **config):
+    async def ping(cls, address, *, timeout=None, pool_config=None):
```

Comment:
You had changed the method signature, but I didn't see any change in methods which calls this one. Is it correct?

Reply:
Yes. No called made use of these config options.